packages/cfd/src/Containers/props.types.ts

```diff
@@ -24,7 +24,7 @@ export type TCFDPersonalDetailsContainerProps = {
     onSubmit: (index: number, value: { [key: string]: string }) => void;
 };
 
-type CFD_Platform = 'dxtrade' | 'mt5' | 'derivez' | 'ctrader';
+export type CFD_Platform = 'dxtrade' | 'mt5' | 'derivez' | 'ctrader';
```

Comment:
Maybe you can refer to TPlatforms namepace in types.ts in wallets package as reference. We let the api do the typing

Reply:
@hirad-deriv better to don't add more technical debt for the future, please update the types here.

Reply:
Updating the types here requires update of at least 10-15 files.
Should I do it ? In my opinion making all of these changes in this PR is out of scope of it